tests/dnp3-toclient-start/test.yaml

```diff
@@ -0,0 +1,19 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+  min-version: 5.0.0
+
+# disables checksum verification, and uses midstream
+args:
+- -k none --set stream.midstream=true --set app-layer.protocols.dnp3.enabled=yes
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: flow
+        app_proto: dnp3
+        flow.pkts_toserver: 2
```

Comment:
Would it be possible to have only 3 packets in the pcap ? So as to have different values for `flow.pkts_toserver` and `flow.pkts_toclient `